tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -848,6 +812,118 @@ def solve_tridiag(
     np.allclose(expected, out)
 
 
+def test_ternary_operator():
+    size = 10
+
+    a = np_as_located_field(IDim)(2 * np.ones((size,)))
+    b = np_as_located_field(IDim)(2 * np.ones((size,)))
+    out = np_as_located_field(IDim)(np.zeros((size,)))
+
+    left = 2.0
+    right = 3.0
+
+    @field_operator
+    def ternary_field_op(
+        a: Field[[IDim], float], b: Field[[IDim], float], left: float, right: float
+    ) -> Field[[IDim], float]:
+        return a if left < right else b
+
+    ternary_field_op(a, b, left, right, out=out, offset_provider={})
+    e = np.asarray(a) + np.asarray(b) if left < right else np.asarray(b)
```

Comment:
Why is there a sum of `a` and `b` here and why is the test passing? Shouldn't it be:
```suggestion
    e = np.asarray(a) if left < right else np.asarray(b)
```
?

Reply:
Because left and right are static such that the right branch is always returned. We could parameterize the test.